packages/ui/src/components/Dropdown/style.less

```diff
@@ -0,0 +1,76 @@
+@import "antd/lib/dropdown/style/index.less";
+@import (reference) "@open-condo/ui/src/tokens/variables.less";
+@import "../style/mixins/typography";
+
+.condo-dropdown {
+  .condo-typography-h5();
+
+  .condo-dropdown-menu {
+    max-height: min(100vh, 360px);
+    padding: @condo-global-spacing-4 0;
+    overflow: auto;
+    border-radius: @condo-global-border-radius-medium;
+    box-shadow: @condo-global-box-shadow-default;
+
+    .condo-dropdown-menu-item {
+      padding: @condo-global-spacing-8 @condo-global-spacing-16;
+    }
+  }
+}
+
+.condo-dropdown.condo-dropdown-wrapper {
+  .condo-dropdown-menu {
+    .condo-dropdown-menu-item-divider {
+      margin: 0;
+    }
+
+    .condo-dropdown-menu-item {
+      padding: @condo-global-spacing-20;
+
+      &:hover {
+        background-color: @condo-global-color-white;
+      }
+
+      .condo-dropdown-item-icon {
+        position: relative;
+        top: 1px;
```

Comment:
?)

Reply:
I forgot to remove it)

Reply:
we have different heights for the text and icon, so we need to move it down by 1 pixel so that it looks even
<img width="487" alt="Снимок экрана 2023-07-31 в 13 26 11" src="https://github.com/open-condo-software/condo/assets/56914444/f849afc0-f312-49fd-b34c-b65572052d0e">
<img width="498" alt="Снимок экрана 2023-07-31 в 13 26 07" src="https://github.com/open-condo-software/condo/assets/56914444/ff3db72e-c98a-48cd-b34b-a0b3a92f882e">